Localization/en-US.hjson

```diff
@@ -0,0 +1,222 @@
+			}
+
+			EskimoArmorTweak: {
+				$parentVal: Eskimo Armor Tweaks
+				Desc:
+					'''
+					Defense increased, immunity to cold water
+					Set bonus: Warmth buff, immunity to Frostburn, Chilled and Frozen
+					'''
+			}
+
+			RainArmorTweak: Rain Armor is now Vanity
+
+			HammerTweaks: {
+				$parentVal: Hammer Tweaks
+				Desc: Rebalances a few hammers/hamaxes.
+			}
+
+			SwatHelmetTweak: {
+				$parentVal: SWAT Helmet Tweaks
+				Desc:
+					'''
+					+14 defense, +15% ranged damage (all types), +5% ranged crit chance
+					If Miscellania is installed, equipping a SWAT Helmet and a Reinforced Vest grants a set bonus of:
+					  Increase ranged damage by 20%
+					  Reduces damage taken by 25%
+					  20% chance to not consume ammo
+					'''
+			}
+
+			FishBowlTweak: {
+				$parentVal: Fish Bowl Tweak
+				Desc: Gives the Fish Bowl +1 defense.
+			}
+
+			BoneBlockFix: {
+				$parentVal: Bone Block Fix
+				Desc: Bone Blocks drop raw Bones when mined.
+			}
+
+			GoldCritterDropTweak: {
+				$parentVal: Gold Critters Drop Coins
+				Desc: Gold critters drop 1 Gold when killed (still sell for 5 Gold when sold).
+			}
+
+			WhoopieCushionTweak: Easter Egg
+
+			VikingHelmetTweak: {
+				$parentVal: Viking Helmet Tweaks
+				Desc: Viking Helmet gives +5% damage when used with Iron or Lead armor.
+			}
+
+			CactusArmorTweak: {
+				$parentVal: Cactus Armor Tweaks
+				Desc: Changes the set bonus of cactus to 25% chance of thorns.
+			}
+
+			MeteorArmorDefenseTweak: {
+				$parentVal: Meteor Armor Defense Tweaks
+				Desc: Defense decreased from 5/6/5 to 4/5/4.
+			}
+
+			PharaohSetTweak: {
+				$parentVal: Pharaoh Set Tweaks
+				Desc: Changes Pharaoh's vanity set to Summoner armor.
+			}
+
+			CrimsonArmorTweak: {
+				$parentVal: Crimson Set Tweak
+				Desc: Crimson Armor now only increases melee damage instead of all damage.
+			}
+
+			UndeadMinerDropRateTweak: {
+				$parentVal: Undead Miner Drop Rate Tweak
+				Desc: Changes the drop rate of mining clothes to a 1/4 chance (1/3 in Expert mode).
+			}
+
+			UndeadMinerRareLifeform: {
+				$parentVal: Undead Miner Rare Lifeform
+				Desc: Makes the Undead Miner show up on the Lifeform Analyzer.
+			}
+
+			FishingPoleTweak: {
+				$parentVal: Fishing Pole Tweaks
+				Desc:
+					'''
+					Sitting Duck Fishing Pole is Hardmode only and the Mechanic's Rod is sold more often
+					If Miscellania is installed, the Mechanic's Rod will also be sold more often.
+					(disable in that mod's config.)
+					'''
+			}
+
+			ExtractSpeedMultiplier: {
+				$parentVal: Extractinator Speed Multiplier
+				Desc: Decreases the amount of time it takes to use the Extractionator (higher is faster).
+			}
+
+			JestersArrowCraft: {
+				$parentVal: Jester's Arrows Crafted per Star
+				Desc:
+					'''
+					Jester's Arrows Crafted per Star.
+					Setting this value to 0 will remove the recipe entirely.
+					'''
+			}
```

Comment:
Re-Logic nerfed the recipe by 50%. Maybe we should also change the default value for the recipe tweak from 50 to 25.